frontend/packages/dev-console/src/components/catalog/utils/category-utils.ts

```diff
@@ -0,0 +1,179 @@
+import * as _ from 'lodash';
+
+export const filterSubcategories = (category, item) => {
```

Comment:
At least exported functions in this utils file should have type definitions. It will help us later to use this utils correctly or refactor this code.

Reply:
I know you have defined the types for the Categories in your PR properly. That's why I've left it here.